react/components/InputCurrency/index.js

```diff
@@ -65,17 +74,13 @@ InputCurrencyWithRef.propTypes = {
   size: PropTypes.string,
   defaultValue: PropTypes.number,
   value: PropTypes.number,
-  /** Currency symbol used as the input prefix. */
-  currencySymbol: PropTypes.string,
-  /** Character used for decimal separation. Eg: '.' in US$ 1,325.25. */
-  decimalSeparator: PropTypes.string,
-  /** Character used for thounsand separation. Eg: ',' in US$ 1,325.25. */
-  thousandSeparator: PropTypes.string,
+  locale: PropTypes.string,
```

Comment:
I would keep `locale` and `currencyCode` as `isRequired`.

Reply:
And would remove defaultProps

Reply:
✅